app/src/main/java/com/csai/swanton/sftp/ChannelSftpFactory.java

```diff
@@ -6,26 +6,25 @@
 import com.jcraft.jsch.Session;
 
 public final class ChannelSftpFactory {
+
+  // TODO: Figure out credentials
```

Comment:
It's probably best to use certificates - not sure if that's possible with the library you're using, but it'd avoid a hard-coded credentials, which would be ideal.

Reply:
I'll see about doing that

Reply:
I'll see about doing that

Reply:
(Won't be in this commit though)